luxonis_ml/data/augmentations/custom/ocr.py

```diff
@@ -0,0 +1,121 @@
+from typing import Tuple
+import random
+import albumentations as A
+
+from albumentations.core.transforms_interface import ImageOnlyTransform
+
+import numpy as np
+from ..utils import AUGMENTATIONS
+
+
+@AUGMENTATIONS.register_module()
+class OCRAugmentation(ImageOnlyTransform):
+
+    def __init__(self, image_size: Tuple[int, int], is_rgb: bool, is_train: bool):
+        """
+
+        @param image_size: OCR model input shape.
+        @type image_size: Tuple[int, int]
+        @param is_rgb: True if image is RGB. False if image is GRAY.
+        @type is_rgb: bool
+        @param is_train: True if image is train. False if image is val/test.
+        @type is_train: bool
+        """
+        super(OCRAugmentation, self).__init__()
```

Comment:
Le'ts keep it just `super().__init__()`. The arguments in `super` are a relic from python 2.